src/components/Filter/Filter.js

```diff
@@ -0,0 +1,314 @@
+      setCheckedList(prev => [...prev, name]);
+    } else {
+      setCheckedList(prev => prev.filter(item => item !== name));
+    }
+  };
+  const handleCheckCancel = () => {
+    setCheckedList([]);
+  };
+  useEffect(() => {
+    fetch('data/filterData.json')
+      .then(response => response.json())
+      .then(result => {
+        setStudioDataList(result.data);
+      });
+  }, []);
+
+  return (
+    <StyleFilter>
+      <Header>
+        <HeaderDiv>
+          <div>
+            <CloseBtn>
+              <FontAwesomeIcon icon={faXmark} onClick={handleModal} />
+            </CloseBtn>
+          </div>
+          <div>
+            <BoldSpan>필터</BoldSpan>
+          </div>
+          <div />
+        </HeaderDiv>
+      </Header>
+      <Body>
+        <BodyDiv>
+          <H2>가격 범위</H2>
+          <GraySpan>평균 1박 요금은 ₩152,631입니다.</GraySpan>
+          <BodyContainer>
+            <RangeBox>
+              <RangeBoxInner>
+                {studioDataList.map(studios => {
+                  return studios.studios.map(st => (
+                    <RangeDiv
+                      key={st.studios_id}
+                      price={Number(st.price_per_hour) * 0.001}
+                    />
+                  ));
+                })}
+              </RangeBoxInner>
+            </RangeBox>
+            <Slider>
+              <RangeSlider payRange={payRange} setPayRange={setPayRange} />
+            </Slider>
+            <BodyInner>
+              <BodyWrapper>
+                <GraySmallSpan>최저 요금</GraySmallSpan>
+                <BodyWrapperInner>
+                  <FeeDiv>₩</FeeDiv>
+                  <FeeInput
+                    type="text"
+                    name="min"
+                    onChange={e => handleRange(e, 0)}
+                    value={payRange[0]}
+                  />
+                </BodyWrapperInner>
+              </BodyWrapper>
+              <BodyWrapperDiv>
+                <GraySmallSpan>
+                  <FontAwesomeIcon icon={faMinus} />
+                </GraySmallSpan>
+              </BodyWrapperDiv>
+              <BodyWrapper>
+                <GraySmallSpan>최고 요금</GraySmallSpan>
+                <BodyWrapperInner>
+                  <FeeDiv>₩</FeeDiv>
+                  <FeeInput
+                    type="text"
+                    name="max"
+                    onChange={e => handleRange(e, 1)}
+                    value={payRange[1]}
+                  />
+                </BodyWrapperInner>
+              </BodyWrapper>
+            </BodyInner>
+          </BodyContainer>
+        </BodyDiv>
+        <BodyDiv>
+          <H2>공간 유형</H2>
+          {ROOM_TYPE.map(type => {
+            return (
+              <BodyTypeDiv key={type.id}>
+                <BodyTypeFilter>
+                  <CheckboxInput
+                    type="checkbox"
+                    name={type.name}
+                    id={type.id}
+                    checked={checkedList.includes(type.name) ? true : false}
```

Comment:
```suggestion
                    checked={checkedList.includes(type.name)}
```
조건 자체가 boolean 값을 반환하기 때문에 위와 같이 작성해도 되겠네요!